data/courses.yaml

```diff
@@ -701,23 +701,3 @@
     Monads, CIC, Subtyping, Tactics, Logic, Sets, Computability, Termination,
     Metaprogramming, Macros, Elaboration, Type Unification, and relevant works
     in other proof assistants.
-- name: Introduction to Cryptography
-  instructor: Matthew Robert Ballard
-  institution: University of South Carolina
-  website: https://587.f24.matthewrobertballard.com
-  tages: ['computer science', 'mathematics', 'cryptography']
```

Comment:
The error seems to be here: this `tages` should have been `tags`

Reply:
Good catch!